src/Configuration/MenuItemConfiguration.php

```diff
@@ -3,6 +3,7 @@
 namespace LAG\AdminBundle\Configuration;
 
 use JK\Configuration\Configuration;
+use LAG\AdminBundle\Utils\StringUtils;
```

Comment:
We did smthing like this for array a while back https://github.com/larriereguichet/array-utils
Don't know if it would be interesting to do the same for strings...

Reply:
Yep, good idea.

Reply:
Done